src/app/printer/pdf/pdf.printer.ts

```diff
@@ -128,4 +152,18 @@ export class PdfPrinter implements Printer {
         return (fontSize * 0.35) / 1.5;
     }
 
+    getFontSizeForBoardsOnSymbols(project) {
```

Comment:
I don't get the meaning of this, it seems that default font size according to board is already a field in the [board object](https://github.com/maxcleme/beadifier/blob/master/src/app/model/board/board.model.ts).

Reply:
I didn't read your last comment before review, I get it know. However I think it should be an new fields in the Board object, allowing us to be sure to not forget it once adding new board size.

Reply:
I think this is a good idea. The switch statement feels a bit hacky and changing it in the board configuration felt wrong since we don't know what type of export someone will do until they hit the button. I'll update this by adding a new symbolSize parameter.